images/Dockerfile.thick

```diff
@@ -7,7 +7,7 @@ ADD . /usr/src/multus-cni
 RUN  cd /usr/src/multus-cni && \
      ./hack/build-go.sh
 
-FROM debian:stable-slim
+FROM gcr.io/distroless/base-debian11:latest
```

Comment:
> I don't grok why this test changes and makes it so we hit that error, but we're not seeing it in other e2e tests? That is baffling to me. I'd love to know why.

In the e2e test, multus-ds mounts the CNI binaries provided by the kind node. These binaries are dynamically linked (compiled with `CGO_ENABLED=1`). When these CNI binaries (e.g., `portmap`) are called, they rely on the GLIBC of multus-ds container, which is the GLIBC of base image `gcr.io/distroless/base-debian11:latest`.

Perhaps change to `debian:stable-slim` allows the e2e test to pass, but it does not meet @s1061123's suggestions.

The reason I've explained so much is simply to convey that the failure of this e2e test is solely an issue with the CI job, not a problem with the changes made in this PR.